.prettierignore

```diff
@@ -1,5 +1,6 @@
 # Don’t try and format the asset catalogue JSON files, which are managed by Xcode
 *.xcassets/
+README.md
```

Comment:
Why?

Reply:
It's bc of [this](https://github.com/ably/ably-chat-swift/actions/runs/12088239382/job/33711338871) linting error

It doesn't give an exact explanation what's wrong and its suggestion running `npx prettier README.md --write` does nothing. Any ideas?

Reply:
done

Reply:
✅